roles/pulp_devel/templates/pulp_cli.bashrc.j2

```diff
@@ -0,0 +1,22 @@
+# set up autocompletion for pulp command
```

Comment:
Think you need to drop the j2 suffix and move that file to the files subdir, as it is not a template (, and certainly not used with the `template:` module).